codespan-reporting/tests/term.rs

```diff
@@ -540,3 +540,41 @@ mod unicode {
     test_emit!(rich_no_color);
     test_emit!(short_no_color);
 }
+
+mod unicode_spans {
+    use super::*;
+
+    lazy_static::lazy_static! {
+        static ref TEST_DATA: TestData<'static, SimpleFile<&'static str, String>> = {
+            let moon_phases = format!("{}", r#"🐄🌑🐄🌒🐄🌓🐄🌔🐄🌕🐄🌖🐄🌗🐄🌘🐄"#);
```

Comment:
HAHAH love these cows

Reply:
Might need to look at cleaning up the formatting here (trailing commas and spacing) - rustfmt seems to ignore `lazy_static!` sadly :(

Reply:
I think i made this match the others.